chainerx_cc/chainerx/routines/explog.cc

```diff
@@ -123,6 +123,27 @@ Array Log10(const Array& x) {
     return out;
 }
 
+Array Log2(const Array& x) {
```

Comment:
Could you add a declaration in `routines/explog.h`?

Reply:
Sorry. I added.